src/create_disk.sh

```diff
@@ -10,8 +10,8 @@
 # an issue and we can discuss configuration needs.
 set -euo pipefail
 
-if [ "$#" -ne 6 ]; then
-	echo 'create_disk <device> <ostree-repo> <ostree-ref> <grub-script> <os-name> <space separated kargs>'
+if [ "$#" -ne 7 ]; then
+	echo 'create_disk <device> <ostree-repo> <ostree-ref> <ostree-remote> <grub-script> <os-name> <space separated kargs>'
```

Comment:
This is getting increasingly awkward...I understand why this was shell script before but it feels like it should probably be a real programming language.

Reply:
I don't disagree. :)

Reply:
(obviously not a blocker for this PR)

Reply:
Agreed not in this PR. I'm on the fence in general. I agree the arg parsing is getting awkward, but so is making a bunch of `exec`s in languages that aren't shell/bash. This does have the advantage of being very clear what inputs it expects.

Then again with things like https://github.com/coreos/coreos-assembler/issues/377 it's going to get worse.